package.json

```diff
@@ -129,7 +129,7 @@
     "nyc": "^11.4.1",
     "prettier-eslint-cli": "^4.7.1",
     "schema-utils": "^0.4.5",
-    "webpack": "^4.0.1",
+    "webpack": "^4.1.1",
```

Comment:
This is going to [conflict](https://github.com/webpack/webpack-cli/pull/319/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R132) with #319.

Reply:
A rebase will fix it :) not a big problem